HostLibraryTests/CMakeLists.txt

```diff
@@ -142,30 +143,39 @@ set(test_sources ${test_sources}
 if(TENSILE_USE_LLVM)
     find_package(LLVM REQUIRED CONFIG)
 
-    set(test_sources ${test_sources}
-        ContractionLibraryLoading_test.cpp
```

Comment:
Are these three tests no longer needed?

Reply:
They were duplicates.

Reply:
Reverting this change to keep PR more narrowly scoped.